### ISSUE TITLE
Remove emulated test macros

### ISSUE THREAD

**Issue by ggwpez** [polkadot-fellows/runtimes#363]
TODO from here:  

https://github.com/polkadot-fellows/runtimes/blob/8ec28f96eeb30fbba30d29006d75e1a3fa1cea1c/integration-tests/emulated/helpers/src/lib.rs#L31-L33

cc @bkontur

**Comment by bkontur** [comment]
yes, I have also TODO [here](https://github.com/polkadot-fellows/runtimes/issues/186),
> - [ ]  remove helpers module from inegrations-tests and re-use from polkadot-sdk

We need to move those macros into the `polkadot-sdk` first. For example, `macro_rules! test_chain_can_claim_assets` and the test case for claiming assets are not even implemented in `polkadot-sdk`.

If we want to fix this for version 1.13, we would need to backport it to both 1.13 and 1.14.

So, I would say this is not a high priority and we can live with this TODO for now. The tests are working for fellows, which is important. We will eventually fix this here: [#4892](https://github.com/paritytech/polkadot-sdk/issues/4892) (good first issue or for whoever wants to contribute).

**Comment by ggwpez** [comment]
> So, I would say this is not a high priority and we can live with this TODO for now

Yes for sure. I just like to open issues for TODOs in the code 😄 so that we dont forget them.

**Comment by programskillforverification** [comment]
@bkontur I already move all macros into the `polkadot-sdk`. Once you remove macro, the third task in  [Backport integration tests for claim assets to the polkadot-sdk from polkadot-fellows repo #4892](https://github.com/paritytech/polkadot-sdk/issues/4892) can be done.
